client/coral-embed/src/Stream.js

```diff
@@ -1,3 +1,5 @@
+/* global __webpack_public_path__ */ // eslint-disable-line no-unused-vars
```

Comment:
What's this line for?

Reply:
Eslint complains that it’s undefined, the `__${prop}__` is global.